float32 sbjspeed
float32 relspeed
float32 distance
uint8 controlstate
float32 acel_cal
float32 acel_temp
float32 e_distance
float32 thwi
float32 ttci
float32 mode
float32 virtualspeed
float32 acel_esp
float32 THW_set
float32 acel_raw
float32 speed_p
float32 speed_i
float32 acel_p
float32 acel_i
float32 dt
